integration_tests/backend/test_reads.py

```diff
@@ -496,29 +496,33 @@ def test_endpoint_node_artifact_get(self):
             assert sum(all_output_counts) == len(all_output_counts) - 1
             assert set(all_output_counts) == set([0, 1])
 
-    # def test_endpoint_node_artifact_result_content_get(self):
-    #     flow_id, n_runs = self.flows["flow_with_multiple_operators"]
-    #     flow = self.client.flow(flow_id)
-    #     workflow_resp = flow._get_workflow_resp()
-    #     dag_id = workflow_resp.workflow_dag_results[0].workflow_dag_id
-    #     dag_result_id = workflow_resp.workflow_dag_results[0].id
+    def test_endpoint_node_artifact_result_content_get(self):
+        flow_id, _ = self.flows["flow_with_multiple_operators"]
```

Comment:
I've been checking for the flows `flow_with_metrics_and_checks` and `flow_with_multiple_operators` (see `test_endpoint_node_artifact_get`). If adding `flow_with_metrics_and_checks` is a straightforward extension, would like it if you add that too.